modules/cloud-router-connection/README.md

```diff
@@ -16,6 +16,416 @@ and to learn how to procure your own Client_Id and Client_Secret follow the link
 [Equinix Fabric Developer Portal](https://developer.equinix.com/docs?page=/dev-docs/fabric/overview)
 <!-- End Module Docs -->
```

Comment:
Same thing with example README.md's. We are replacing and need to delete the previous sections. Remove all the `<!-- Begin Module Docs (Do not edit contents) --><!-- End Module Docs -->` sections (including their contents). Should be done in all module README.md files.

Reply:
Addressed with [commit](https://github.com/equinix/terraform-equinix-fabric/pull/102/commits/4e7ee85646d9482c993afb4565814c3b383a556b)